dist/profile/manifests/pkgrepo.pp

```diff
@@ -90,7 +91,7 @@
     require     => File[$repos],
   }
 
-  apache::vhost { $repo_fqdn:
+  © { $repo_fqdn:
```

Comment:
typo?

Reply:
yes ... I usually do not copyright my work 😂